command/harness/dlite/dlite.go

```diff
@@ -103,29 +104,31 @@ func (c *dliteCommand) run(*kingpin.ParseContext) error {
 		cancel()
 	})
 
-	instanceStore, stageOwnerStore, err := database.ProvideStore(c.env.Database.Driver, c.env.Database.Datasource)
-	if err != nil {
-		logrus.WithError(err).Fatalln("Unable to start the database")
-	}
-
-	c.stageOwnerStore = stageOwnerStore
-	c.poolManager = drivers.New(ctx, instanceStore, &c.env)
+	// Initialize metrics
+	c.registerMetrics()
 
-	poolConfig, err := harness.SetupPool(ctx, &c.env, c.poolManager, c.poolFile)
-	defer harness.Cleanup(&c.env, c.poolManager) //nolint: errcheck
+	poolConfig, err := c.setupPool(ctx)
+	defer harness.Cleanup(&c.env, c.poolManager, true, true) //nolint: errcheck
```

Comment:
Do we need to call defer cleanup on the pool manager as well? Only one or the other would be used right?